base_layer/core/src/base_node/state_machine_service/states/horizon_state_sync/horizon_state_synchronization.rs

```diff
@@ -531,6 +532,7 @@ impl<'a, B: BlockchainBackend + 'static> HorizonStateSynchronization<'a, B> {
 
         let mut prev_mmr = 0;
         let mut prev_kernel_mmr = 0;
+        let bitmap = Arc::new(self.db().get_complete_deleted_bitmap_at_tip().await?.into_bitmap());
```

Comment:
1. The db uses `fetch` as the verb, so let's keep it the same
2. The tip could have somehow changed between the call in line 529, so rather pass in the expected tip so that it can fail if the tip has somehow changed
```suggestion
        let bitmap = Arc::new(self.db().fetch_complete_deleted_bitmap_at(header.hash()).await?.into_bitmap());
```